README.md

```diff
@@ -37,14 +38,15 @@ To compile casacore you need to meet the following requirements:
 
 On Debian / Ubuntu you can install these with:
  ``` 
-$ sudo apt-get install cmake gfortran libncurses5-dev libreadline-dev flex \
-    bison libblas-dev liblapacke-dev libcfitsio3-dev wcslib-dev
+$ sudo apt-get install build-essential cmake gfortran g++ libncurses5-dev \
+    libreadline-dev flex bison libblas-dev liblapacke-dev libcfitsio3-dev \
+    wcslib-dev
 ```
 
 and the optional libraries:
 ```
 $ sudo apt-get install libhdf5-serial-dev libfftw3-dev python-numpy \
-    libboost-python-dev
+    libboost-python-dev libopenmpi-dev
```

Comment:
openmp is not the same as openmpi, (shared memory vs message passing)
openmp needs libgomp but on debian based systems that is already a dependency of gcc